### ISSUE TITLE
Using /audiomode on Chrome may immediately freeze Hubs 

### ISSUE THREAD

**Issue by misslivirose** [mozilla/hubs#2414]
**Description**
In Chrome, if you try to use /audiomode, sometimes Chrome will immediately freeze Hubs. It doesn't reproduce 100% of the time but I hit it on my first time testing after it was reported by someone else in the room. 

**To Reproduce**
1. Enter a room
2. Try to change the /audiomode node

Some of the time, this causes Hubs to freeze entirely 

**Expected behavior**
Hubs does not freeze

**Hardware** 
 - Device: Desktop
 - OS: Windows
 - Browser Chrome (no plugins, version 81.0.4044.122)

**Additional context**
Logs: 

```
three.module.js:42428 The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu
getContext @ three.module.js:35178
console.js:35 Batching must be forced on with forceMeshBatching or forceImageBatching. Disabling batching.
e.<computed> @ react.production.min.js:22
console.js:35 Hub ID: BQwxTJd
console.js:35 Reticulum @ jolly-sorcerer.reticulum.io: v1.0.20200423002053 on earth
console.js:35 Phoenix Socket URL: wss://jolly-sorcerer.reticulum.io:443
console.js:35 Logged into account 352265881025249397
console.js:35 THREE.GLTFLoader: Duplicating UVs to support aoMap.
console.js:35 Janus host: lucid-ardent.reticulum.io:443
console.js:35 Scene URL: https://uploads-prod.reticulum.io/files/5bb055ff-f1f0-4098-9211-7db11439c02c.bin
hub.js:1427 The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu
(anonymous) @ hub-282484949123b7ac387e.js:1
console.js:35 Deprecated API usage: PDFDocumentLoadingTask.then method, use the `promise` getter instead.
2console.js:35 THREE.GLTFLoader: Duplicating UVs to support aoMap.
2console.js:35 Avatar does not an 'allOpen' animation, disabling hand animations
e.<computed> @ react.production.min.js:22
console.js:35 [three-pathfinding]: Use THREE.BufferGeometry, not THREE.Geometry, to create zone.
e.<computed> @ react.production.min.js:22
physics-utils.js:59 traverseMeshesAndAddShapes
console.js:35 heightfield or trimesh found on scene
console.js:35 7e4179e4-2dc2-4f56-852e-5337015b6f67: cancel occupant connection, occupant left during or after webrtcup
e.<computed> @ react.production.min.js:22
console.js:35 THREE.GLTFLoader: Duplicating UVs to support aoMap.
2hubs-proxy.com/https://rhiannanberry.github.io:1 Failed to load resource: the server responded with a status of 404 (Not Found)
3console.js:35 THREE.GLTFLoader: Duplicating UVs to support aoMap.
2console.js:35 Avatar does not an 'allOpen' animation, disabling hand animations
e.<computed> @ react.production.min.js:22
2console.js:35 THREE.GLTFLoader: Duplicating UVs to support aoMap.
console.js:35 Using microphone: Default - Arctis 5 Mic (SteelSeries Arctis 5 Chat) (1038:1250)
raven.js:381 Uncaught TypeError: Cannot read property 'disconnect' of undefined
    at n.tick (audio-settings-system.js:113)
    at HTMLElement.tick (a-scene.js:798)
    at HTMLElement.render (a-scene.js:848)
    at three.module.js:25563
    at r (three.module.js:15024)
    at a (raven.js:377)
DevTools failed to load SourceMap: Could not load content for https://hubs.mozilla.com/color-shift.worker-787a591ca4809b890107.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://hubs.mozilla.com/sketchfab-zip.worker-91fe1d86be396c2ee61e.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://hubs.mozilla.com/gifparsing.worker-ef1abc6384654b227b6a.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://hubs.mozilla.com/ammo.worker-e61f9bb1d8daed2cf27b.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://assets-prod.reticulum.io/assets/js/pdfjs-dist@2.1.266/build/pdf.worker.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE```

**Comment by misslivirose** [comment]
Marking as P2 since /audiomode is experimental, but it is becoming widely used so it may be worth bumping up

**Comment by misslivirose** [comment]
Possibly seeing this in Firefox too

**Comment by jdm** [comment]
#2466 may or may not be related.

**Comment by misslivirose** [comment]
Potentially fixed with #2456

**Comment by misslivirose** [comment]
Closing since we haven't had reports of this in a while - will re-open if we start getting reports that this wasn't fixed with #2456
